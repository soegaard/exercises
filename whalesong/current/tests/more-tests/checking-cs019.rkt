#lang planet dyoo/whalesong/cs019

(define (greet name)
  (string-append "hello " name))

(check-expect (greet "danny") "hello danny")
(check-expect (greet "huh") "this should fail")
(check-expect (greet "world") "hello world")
