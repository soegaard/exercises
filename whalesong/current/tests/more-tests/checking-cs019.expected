Running tests...
check-expect: actual value "hello huh" differs from "this should fail", the expected value
  at: #<path:/gpfs/main/home/dyoo/work/whalesong/tests/more-tests/checking.rkt>, line 7, column 0
Ran 3 tests.
2 tests passed.
One test failed.
